### DISCUSSION TITLE
`isowords` app target missing

### DISCUSSION THREAD

**Discussion by MartinP7r** [pointfreeco/isowords#190] [General]
**Describe the bug**

After following the readme instructions, there is no `isowords.xcodeproj` and the package itself also doesn't contain a `isowords` application target.

```
Open the Xcode project App/isowords.xcodeproj.

To run the client locally, select the isowords target in Xcode and run (⌘R).
```

**To reproduce**
follow the steps in the readme.

**Expected behavior**
Be able to build the `isowords` app standalone

**Screenshots/video**
If applicable, add screenshots/video to help explain the problem.

**Environment**
 - Device: [e.g. iPhone 12 Pro]
 - OS: [e.g. iOS 14]

**Additional context**
Add any more context about the problem here.

**Comment by mbrandonw** [comment]
Hi @MartinP7r, you don’t see this project in your clone of the repo?

https://github.com/pointfreeco/isowords/tree/main/App/isowords.xcodeproj

If not, maybe you should try re-cloning. 

Also this doesn’t seem to be an issue with the app so I am going to convert to a discussion.

**Reply by MartinP7r** [reply]
Hi @mbrandonw, thanks so much for the swift response!  
It actually worked after re-cloning. That was very strange 🤔 
Sorry for the trouble!
